jobs/pacman-data-shipper/src/test/java/com/tmobile/cso/pacman/datashipper/util/UtilTest.java

```diff
@@ -0,0 +1,70 @@
+package com.tmobile.cso.pacman.datashipper.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.HashMap;
+
+public class UtilTest {
+
+    @Test
+    public void testContains() {
+        HashMap<String, String> hash1 = new HashMap<>();
+        HashMap<String, String> hash2 = new HashMap<>();
+        hash2.put("foo", "3");
+
+        Assert.assertFalse(Util.contains(hash2, hash1, new String[]{"foo"}));
+
+        hash1.put("foo", "3");
+        Assert.assertTrue(Util.contains(hash2, hash1, new String[]{"foo"}));
+    }
+
+    @Test
+    public void testConcatenate() {
+        HashMap<String, String> hash = new HashMap<>();
+        hash.put("foo", "3");
+
+        Assert.assertEquals("3",
+                Util.concatenate(hash, new String[]{"foo"}, ","));
```

Comment:
After this assert, do another `hash.put` with other values and then another test

Reply:
Done